compact_str/Cargo.toml

```diff
@@ -11,12 +11,20 @@ readme = "../README.md"
 keywords = ["string", "compact", "small", "memory", "mutable"]
 categories = ["encoding", "parsing", "memory-management", "text-processing"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+to-compact-str-float-spec = ["ryu", "to-compact-str-specialisation"]
+to-compact-str-int-spec = ["itoa", "to-compact-str-specialisation"]
+to-compact-str-specialisation = ["castaway"]
```

Comment:
I would really prefer to not have so many features, especially ones tied to performance, can we make these dependencies required instead of optional?

Reply:
sure

Reply:
Done!